contrib/grinder/scripts/query.py

```diff
@@ -97,11 +97,27 @@ def generate(self, time, logger):
 #    logger(result.getText())
     return result
 
+class EnumSearchQuery(AbstractQuery):
+  query_interval_name = 'enum_search_queries_per_interval'
+  query_name = "EnumSearchQuery"
+  test_number = 6
```

Comment:
@VinnyQ The single plot query is 3 in this file, which is probably clashing with your test number.